.github/workflows/cypress-tests-runner.yml

```diff
@@ -0,0 +1,175 @@
+        uses: baptiste0928/cargo-install@v2.2.0
+        with:
+          crate: diesel_cli
+          features: postgres
+          args: "--no-default-features"
+
+      - name: Diesel migration run
+        if: ${{ ((github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name)) || (github.event_name == 'merge_group')}}
+        shell: bash
+        env:
+          DATABASE_URL: postgres://db_user:db_pass@localhost:5432/hyperswitch_db
+        run: make migrate locked-schema=yes
+
+      - name: Install Cypress
+        run: |
+          cd cypress-tests
+          npm install
+
+      - name: Build project
+        if: ${{ ((github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name)) || (github.event_name == 'merge_group')}}
+        run: cargo build --package router --bin router
+
+      - name: Setup Local Server
+        if: ${{ ((github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name)) || (github.event_name == 'merge_group')}}
+        run: |
+          # Start the server in the background
+          target/debug/router &
+          SERVER_PID=$!
+
+          # Wait for the server to start in port 8080
+          COUNT=0
+          while ! nc -z localhost 8080; do
+            if [ $COUNT -gt 12 ]; then # Wait for up to 2 minutes (12 * 10 seconds)
+              echo "Server did not start within a reasonable time. Exiting."
+              kill ${SERVER_PID}
+              exit 1
+            else
+              COUNT=$((COUNT+1))
+              sleep 10
+            fi
+          done
+
+      - name: Run Cypress tests
+        if: ${{ ((github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name)) || (github.event_name == 'merge_group')}}
+        env:
+          CYPRESS_BASEURL: "http://localhost:8080"
+        shell: bash
+        run: |
+          cd cypress-tests
+
+          RED='\033[0;31m'
+          RESET='\033[0m'
+
+          declare -A failed_connectors  # Associative array to track failed connectors
+
+          for i in $(echo "${CONNECTORS}" | tr "," "\n"); do
+            echo "${i}"
+            for j in "payments" "payouts"; do
+              if ! CYPRESS_CONNECTOR="${i}" npm run cypress:"${j}"; then
+                failed_connectors["${i}"]=1
+              fi
+            done
+          done
+
+          if [ ${#failed_connectors[@]} -gt 0 ]; then
+            echo -e "${RED}One or more connectors failed to run:${RESET}"
+            echo "${!failed_connectors[@]}"
+            exit 1
+          fi
+
+      - name: Upload Cypress test results
```

Comment:
Uploading artifacts seems to have failed in the latest workflow run, could you check why it failed?

Reply:
yes, it cannot determine the path even though exact path have been passed. needs debugging a bit.